src/main/java/dev/jbang/dependencies/DependencyUtil.java

```diff
@@ -88,7 +88,7 @@ public static ModularClassPath resolveDependencies(List<String> deps, List<Maven
 									.map(JitPackUtil::ensureGAV)
 									.collect(Collectors.toList());
 		// And if we encountered URLs let's make sure the JitPack repo is available
-		if (!depIds.equals(deps) && !repos.stream().anyMatch(r -> REPO_JITPACK.equals(r.getUrl()))) {
+		if (!depIds.equals(deps) && repos.stream().noneMatch(r -> REPO_JITPACK.equals(r.getUrl()))) {
```

Comment:
Why change to this ? The original could end early where as the new one needs to always iterate through the full list ?

Reply:
Dropped